src/ansys/geometry/core/designer/designpoint.py

```diff
@@ -28,6 +28,8 @@
 from ansys.geometry.core.misc.units import UNITS
 
 if TYPE_CHECKING:  # pragma: no cover
+    import pyvista as pv
```

Comment:
If we are going to import PyVista for typechecking, can we import it for the whole module and avoid the lazy import in `_to_polydata` method? What do you think?

Reply:
Hmm I don't see it. TYPE_CHECKING is only rendered at IDE level. Not when running the code. Which means that it does not really get imported. I would still leave it where it is :)